tests/test_utils.py

```diff
@@ -30,22 +30,22 @@ def setUp(self):
     def test_polyline5_2d_decoding(self):
 
         decoded = [(8.68864, 49.42058), (8.68092, 49.41578)]
-        self.assertEqual(decoded, utils.decode_polyline5(self.coords2d_5prec))
+        self.assertEqual(decoded, utils.decode_polyline5(self.coords2d_5prec, order="latlng"))
```

Comment:
ah sorry, must've been a misunderstanding: since this is quite confusing, can we change the encoded strings in the `setUp`?